class1.md

```diff
@@ -0,0 +1,261 @@
+
+## Concepts in Machine Learning Class 1: Introduction to Machine Learning
+
+### Objectives
+
+Welcome to Concepts in Machine Learning from fredhutch.io! This course is designed for researchers who are interested in machine learning, and assumes no prior programming or statistics experience. There are no prerequisites for this course.
+
+This course will introduce you to machine learning and give a broad overview of supervised and unsupervised machine learning methods. The goal of this course is to provide some conceptual background for our Machine Learning in R and Python series. Further, researchers who complete this course should have an understanding of how machine learning can be applied in biology research.
```

Comment:
introduce you to machine learning and focus specifically on supervised and unsupervised machine learning methods.